CONTRIBUTING.md

```diff
@@ -7,6 +7,9 @@ If you would like to make a significant change, it's a good idea to first open a
 ### Making the request
 Development takes place against the dev branch of this repository and pull requests should be opened against that branch.
 
+### Code Style
+This project follows the [Google Java Styleguide](https://google.github.io/styleguide/javaguide.html), and this style is enforced as part of the `check` task. We recommend you install [the `google-java-format` plugin for your IDE](https://github.com/google/google-java-format), or use the `gradle spotlessApply` task to format code before checking in.
```

Comment:
How do we ensure that the style has been applied? Or are you saying that the check will run as part of the build and fail if the style is not preserved but it won't apply fixes?